Gemfile

```
@@ -2,74 +2,38 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.2.2"
-
-# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
 gem "rails", "~> 7.0.7", ">= 7.0.7.2"
```

バージョンアップ時に特定の作業が必要だったりするものは固定しましょう。

```suggestion
gem "rails", "7.0.7.2"
```